resolwe/flow/utils/test.py

```diff
@@ -224,6 +224,8 @@ def tearDown(self):
                 shutil.rmtree(data_dir, ignore_errors=True)
                 shutil.rmtree(export_dir, ignore_errors=True)
 
+        Process.objects.all().delete()
```

Comment:
Problem is that `_register_schemas` caches processes (with contributor relation), but admin id is increased in each test run, so it cannot link them. Maybe this is new in Django 1.10.

I've also noted that super-user is created in `setUp` and in `_register_schemas` (only if it doesn't exist), so you can probably remove the latest.